src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/repair/soraldbot/SoraldCLAdapter.java

```diff
@@ -0,0 +1,201 @@
+package fr.inria.spirals.repairnator.process.step.repair.soraldbot;
+
+import org.apache.commons.io.FileUtils;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import sorald.Constants;
+import sorald.Main;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import static fr.inria.spirals.repairnator.process.step.repair.soraldbot.SoraldConstants.SORALD_GIT_PATCHES_DIR;
+
+public class SoraldCLAdapter {
```

Comment:
What does 'CL' means? Looks very close to SoraldCIAdapter? Could we remove to avoid confusion?